target/linux/ipq806x/base-files/lib/upgrade/platform.sh

```diff
@@ -3,6 +3,10 @@
 PART_NAME=firmware
 REQUIRE_IMAGE_METADATA=1
 
+get_magic_long_vr2600v() {
```

Comment:
Well, not all additions to this files. Only the additions to platform_check_image() and your get_magic_long_vr2600v functions. Please update to the latest LEDE to test a sysupgrade afterwards.